src/modules/storage/LedgerStorage.ts

```diff
@@ -2412,26 +2413,23 @@ export class LedgerStorage extends Storages {
      * of the returned Promise is called with the records
      * and if an error occurs the `.catch` is called with an error.
      */
-    public getValidatorsAPI(
-        height: Height | null,
-        address: string | null,
-        conn: mysql.PoolConnection | undefined = undefined
-    ): Promise<any[]> {
+    public getValidatorsAPI(height: Height | null, address: string | null, conn: mysql.PoolConnection | undefined = undefined, limit?: number, page?: number): Promise<any[]> {
```

Comment:
Don't change the parameters of this function.
It is used in other services.

Reply:
These are optional paramters, I don't think these parameters will effect the functionality

Reply:
If page, pageSize is not defined, it is set to the default value.
page = 1 , pageSize = 30
This affects all services that use this function.
It is limited to 30 due to paging to the service that inquires the entire validators.

Reply:
Now I have implemented additional check, If there is pagination in the request then it return according to the request, if there is no pagination in the request, it will return all the validators

Reply:
Now you can get all validators, if you hit /validators without page and pageSize